### DISCUSSION TITLE
python wrappers for LAMMPS from brew install

### DISCUSSION THREAD

**Discussion by xiansch** [Homebrew/discussions#2914] [Everyday usage]
Hi everyone, hope this is a good place to ask.  I installed LAMMPS molecular dynamics code on my M1 Mac with homebrew and it works fine.  Now I am wondering if it also possible to install the python wrappers for LAMMPS as well, as detailed in https://docs.lammps.org/Python_install.html.  The references to how to get the python wrappers in the LAMMPS tutorials reference using CMake, but I'm not sure how to do the same thing with the homebrew install.  (My executables are in /opt/homebrew/Cellar/lammps/20210929-update2/ and there are no makefiles.). Asking the LAMMPS folks, they referred me to ask the homebrew people.

**Comment by carlocab** [comment]
It should be possible to modify the formula to install the Python bindings as well. The simplest way might be to migrate the existing `install` method to use CMake instead, though.

I don't use lammps, so I'm not in a good position to determine whether changes made to the installation preserves previously existing functionality (since some users might be relying on them), but perhaps you'd like to give it a try and then open a pull request for it.
